vulnerabilities/importers/nginx.py

```diff
@@ -0,0 +1,190 @@
+# Copyright (c)  nexB Inc. and others. All rights reserved.
+# http://nexb.com and https://github.com/nexB/vulnerablecode/
+# The VulnerableCode software is licensed under the Apache License version
+# Data generated with VulnerableCode require an acknowledgment.
+#
+# You may not use this software except in compliance with the License.
+# You may obtain a copy of the License at: http://apache.org/licenses/LICE
+# Unless required by applicable law or agreed to in writing, software dist
+# under the License is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+# CONDITIONS OF ANY KIND, either express or implied. See the License for t
+# specific language governing permissions and limitations under the Licens
+#
+# When you publish or redistribute any data created with VulnerableCode or
+# derivative work, you must accompany this data with the following acknowl
+#
+#  Generated with VulnerableCode and provided on an 'AS IS' BASIS, WITHOUT
+#  OR CONDITIONS OF ANY KIND, either express or implied. No content create
+#  VulnerableCode should be considered or used as legal advice. Consult an
+#  for any legal advice.
+#  VulnerableCode is a free software  from nexB Inc. and others.
+#  Visit https://github.com/nexB/vulnerablecode/ for support and download.
+
+import asyncio
+import dataclasses
+import json
+
+import requests
```

Comment:
Nit: we should sort imports. But we should also switch to using black and isort, so we can do it all at once!